trane/datasets/data/store/LICENSE

```diff
@@ -0,0 +1,340 @@
+		    GNU GENERAL PUBLIC LICENSE
```

Comment:
Add this file to here:
https://github.com/trane-dev/Trane/blob/main/pyproject.toml#L69